app/assets/stylesheets/modules/_family.scss

```diff
@@ -128,6 +124,83 @@
       background-size: 100%;
       display: inline-block;
       height: em(210);
+      margin-top: em(40);
+  }
+}
+
+.family-names {
+  text-align: center;
```

Comment:
Properties should be ordered background-color, border-radius, margin, padding, text-align